regular_expressions_basics/regular_expressions_basics.md

```diff
@@ -0,0 +1,379 @@
+<!--
+
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 1.0.0
+language: en
+narrator: UK English Female
+mode: Textbook
+previous_sequential_module: demystifying_regular_expressions
+
+title: Regular Expressions Basics
+
+comment:  Begin to use regular expressions, or regex, for simple pattern matching.
+
+long_description: Regular expressions, or regex, are a way to specify patterns (such as the pattern that defines a valid email address, medical record number, or credit card number).  Learn to compose basic regular expressions in order to find and use important data.
+
+estimated_time_in_minutes: 60
+
+@pre_reqs
+Learners should have some knowledge about patterns in biomedical data and understand the utility of regular expressions (regex).  For an introduction to these concepts, consider the [Demystifying Regular Expressions](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_regular_expressions/demystifying_regular_expressions.md#1) module.
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Define a simple alphanumeric pattern in regex notation
+- List common ranges and character groups in regex
+- Quantify characters appearing optionally, once, or multiple times in regex
+- Test regular expressions in a browser
+@end
+
+@version_history 
+No previous versions.
+@end
+
```

Comment:
More metadata!

I propose a new `regex` sequence since these really are one module split into 3 parts.

```suggestion
good_first_module: false
coding_required: true
coding_level: basic
sequence_name: regex
previous_sequential_module: demystifying_regular_expressions
@sets_you_up_for

@end 
@depends_on_knowledge_available_in
- demystifying_regular_expressions
@end
```